pontoon/base/urls.py

```diff
@@ -30,6 +30,15 @@
     url(r'^contributor/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
         RedirectView.as_view(url="/contributors/%(email)s/", permanent=True)),
 
+    # List all active localization teams
+    url(r'^teams/$',
+        views.locales,
+        name='pontoon.teams'),
+
+    # Redirect to a team page
+    url(r'^teams/(?P<locale>[A-Za-z0-9\-\@\.]+)/$',
```

Comment:
The `/teams/sl` URL is only there if localizers type it in the URL by chance. They might be expecting it because it follows the same pattern as `/projects/x` and `/contributors/y`. Also, it doesn't hurt.

The reasons I went with the `/sl` URLs instead of only `/teams/sl` are:
* It allows us to removing chunks from the translate URLs `/sl/project/path` and never 404.
* Users of Pootle and Verbatim are familiar with it.
* It's shorter. :-)

We didn't have any internal links to `/sl`, so localizers don't even know these pages exists and are asking for them (which is also why we need the `/teams` page). But there are blog posts linking to them.